### PR TITLE
[NCL-2814]: Make the target group for promotion configurable

### PR DESCRIPTION
**Pull request by jbartece**
Defaults to builds-from-pnc

### CODE REVIEW

File: maven-repository-manager/src/main/java/org/jboss/pnc/mavenrepositorymanager/MavenRepositorySession.java [hunk 1]
```diff
@@ -73,7 +73,6 @@
  * group, to expose them for use in successive builds in the chain.
  */
 public class MavenRepositorySession implements RepositorySession {
-
```
Comment (matejonnet) [inline maven-repository-manager/src/main/java/org/jboss/pnc/mavenrepositorymanager/MavenRepositorySession.java]: Please add empty line back as there is always an empty line after the class line.

Comment (jbartece) [inline maven-repository-manager/src/main/java/org/jboss/pnc/mavenrepositorymanager/MavenRepositorySession.java]: DONE

**Comment by pkocandr** [comment]
I don't like the field name promoteBuildToGroup, it is not very clear what that is. I think buildPromotionGroup would be more understandable. Apart from that it looks ok.

**Comment by jbartece** [comment]
OK. I'll rename it

**Comment by pkocandr** [comment]
OK, now it's perfect ;-)

**Comment by alexcreasy** [comment]
@jbartece UI changes work fine :)

**Pull request merged**